spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './docs/menu'
+
+describe Menu do
+  it 'responds to Menu class' do
```

Comment:
Don't think this test is needed. Rspec always initialises the subject to be a new instance of the described class